packages/components/package.json

```diff
@@ -36,6 +36,7 @@
 		"@emotion/serialize": "^1.0.2",
 		"@emotion/styled": "^11.6.0",
 		"@emotion/utils": "1.0.0",
+		"@types/react-dates": "17.1.10",
```

Comment:
So far we haven't had to add any `@types/*` dependency directly to the components package's `package.json` file — have you tried adding it only to the root `package.json`?

Reply:
Oops. I always get confused about what goes where.

Reply:
I believe we should only list non-dev dependencies for a package (although I'm not 100% sure either)